LEAF_Request_Portal/api/controllers/GroupController.php

```diff
@@ -57,10 +57,7 @@ public function post($act)
         $this->index['POST'] = new ControllerMap();
 
         $this->index['POST']->register('group', function ($args) use ($group) {
-            return $group->addGroup(
-                XSSHelpers::sanitizeHTML($_POST['groupName']),
-                XSSHelpers::sanitizeHTML($_POST['groupDesc'])
-            );
+            return $group->addGroup($_POST['title']); // POST for title of group
```

Comment:
If using anything from a POST, it should be sanitized before anything done with it.  I looked at the function used, and that function doesn't seem to do this.

Reply:
I added the sanitize to the correct function that was using HTML POSTS